Sources/ConcurrencyHelpers/MakeStream.swift

```diff
@@ -0,0 +1,25 @@
+/// `makeStream` is part of Swift 5.9
+/// https://github.com/apple/swift-evolution/blob/main/proposals/0388-async-stream-factory.md
+public func makeStream<T>(of _: T.Type) -> (AsyncStream<T>, AsyncStream<T>.Continuation) {
```

Comment:
Why not have the same signature as `AsyncStream.init()` ?

Reply:
They have the same signature as they will in Swift 5.9 according to this proposal:

https://github.com/apple/swift-evolution/blob/main/proposals/0388-async-stream-factory.md#proposed-solution

Or what do you mean?

Reply:
In detailed design section:
```swift
public static func makeStream(
      of elementType: Element.Type = Element.self,
      bufferingPolicy limit: Continuation.BufferingPolicy = .unbounded
  ) -> (stream: AsyncStream<Element>, continuation: AsyncStream<Element>.Continuation) {
```

Reply:
Mm, I see now, let me update.